### PR TITLE
Add county/state/country population GET requests

### LINKED ISSUE #21: Get population data from API
### User Story
As a dev, I want to retrieve population data at the county level from an API, so I can use it in statistics shown to the user

### Additional Context
State and Country too if easy enough

### Acceptance Criteria
- [x] Retrieve county population data

### PR DESCRIPTION
**Pull request by dansjack**
## Summary
Closes #21 

Adds three more requests
* getCountyPopulation()
* getStatePopulation()
* getCountryPopulation()

The data comes from the [U.S. Census Bureau's 2019 population estimate API](https://www.census.gov/data/developers/data-sets/popest-popproj/popest.html)

I've added a file with all the FIPS codes in it to /assets (that's where 3222 of my additions are coming from), but it might be better to put this data in the database. 

## Checklist
- [ ] New tests added to account for changes in this MR
- [x] All tests passing locally
- [x] All tests passing on CircleCI
- [ ] Assigned at least 2 reviewers

### CODE REVIEW

File: app/src/main/java/com/nsc/covidscore/api/APIHelpers.java [hunk 1]
```diff
@@ -9,55 +10,134 @@
...
 
 public class APIHelpers {
     public static void handleResponse(
             String type, String response, String county, String state, VolleyJsonCallback cb) {
         try {
-            boolean found = false;
-            if (type.equals(Constants.COUNTY)) {
-                JSONArray counties = new JSONArray(response);
-                if (counties.length() > 1) {
+            switch (type) {
```
Comment (unicornsleuth) [inline app/src/main/java/com/nsc/covidscore/api/APIHelpers.java:20]: Good call using switch/case, always a better choice IMO

**Review by unicornsleuth — APPROVED** [review]
Great! Code all looks neat and is pretty easy to understand, and it works as expected!

**Review by AllisonBC — APPROVED** [review]
Wow, great work!

**Review by chonger878 — APPROVED** [review]
App opened with no error except for some warnings about a no-arg constructor in the build message.  I take it that won't be an issue, no?  Other than that, fantastic detail and work! :)

**Pull request merged**